src/utils.js

```diff
@@ -18,10 +18,12 @@ export function addToDom(node, canvas) {
   node.setAttribute('data-kontra', '');
   if (container) {
     let target =
-      container.querySelector('[data-kontra]:last-of-type') || canvas;
-    container.insertBefore(node, target.nextSibling);
+      [
+        ...container.querySelectorAll(':scope > [data-kontra]')
```

Comment:
`last-of-type` doesn't work on any selector, it only matches the last node name. So if the dom is

```
div.foo
div.foo
div
```

trying to do `.foo:last-of-type` doesn't work as `:last-of-type` first matches the last `div` of the group _then_ checks if it has the `.foo` class (which it doesn't so returns `null`)